extensions/amp-analytics/0.1/storage-impl.js

```diff
@@ -135,7 +135,11 @@ export class Storage {
           const blob = btoa(JSON.stringify(store.obj));
           return this.binding_.saveBlob(this.origin_, blob);
         })
-        .then(this.broadcastReset_.bind(this));
+        .then(this.broadcastReset_.bind(this))
+        .catch(reason => {
```

Comment:
Catching after the rebroadcast has two side effects:

- If the `#saveBlob` fails, we'll never `#broadcastReset`.
- Otherwise If the `#broadcastReset` throws, we're catching it, too.